pkg/proxy/ipvs/graceful_termination.go

```diff
@@ -150,13 +150,13 @@ func (m *GracefulTerminationManager) GracefulDeleteRS(vs *utilipvs.VirtualServer
 	if err != nil {
 		return err
 	}
-	klog.V(5).Infof("Adding an element to graceful delete rsList: %+v", ele)
+	klog.V(5).InfoS("Adding an element to graceful delete rsList", "element", ele)
```

Comment:
Why 10 lines before you used same object `ele`, but under key "realServer" and used `.String()`. Goal of migration is not only to a `s/Infof/InfoS`, but also to make sure that logs have some minimum quality and follow logging guidelines.

Reply:
I've improved the logging and made it consistent, also we do not need to use .String() as we internally use 
```
switch v.(type) {
		case string, error:
			b.WriteString(fmt.Sprintf("%s=%q", k, v))
		case []byte:
			b.WriteString(fmt.Sprintf("%s=%+q", k, v))
		default:
			if _, ok := v.(fmt.Stringer); ok {
				b.WriteString(fmt.Sprintf("%s=%q", k, v))
			} else {
				b.WriteString(fmt.Sprintf("%s=%+v", k, v))
			}
		}
```
logging output using rs and rs.String(), adding the outputs in separate comment